frontend/app/lib/presets.ts

```diff
@@ -139,7 +139,7 @@ export const availableTools = [
     title: 'Widget',
     image: widgetImg,
     bgColor: 'from-wm-red to-wm-red-fade',
-    link: 'create/widget',
+    link: 'widget',
```

Comment:
I'm a little confused. Even before this PR, https://webmonetization.org/tools/banner works, and it isn't https://webmonetization.org/tools/create/banner. Why does the current thing work even?

Reply:
Yes, why does https://webmonetization.org/tools/banner work already (even before this PR), including the links from home page (https://webmonetization.org/tools/)